ext/nnstreamer/tensor_filter/tensor_filter_llamacpp.cc

```diff
@@ -452,22 +434,13 @@ TensorFilterLlamaCpp::invoke_dynamic (GstTensorFilterProperties *prop,
   prompt.erase (prompt.find_last_not_of (" \t\n\r\f\v") + 1);
 
   if (prop->invoke_async) {
-
-    if (prop->invoke_async_callback == nullptr) {
-      throw std::invalid_argument ("invoke_async_callback is null");
-    }
```

Comment:
In asynchronous mode (invoke_async=true), the existence of the callback function (invoke_async_callback) is not checked. Is this okay?